README.md

```diff
@@ -557,6 +559,15 @@ docker run -d -p 80:8080 -v host/path/to/config.json:/src/Parse-Dashboard/parse-
 
 If you are not familiar with Docker, ``--port 8080`` will be passed in as argument to the entrypoint to form the full command ``npm start -- --port 8080``. The application will start at port 8080 inside the container and port ``8080`` will be mounted to port ``80`` on your host machine.
 
+# Features
+## Browse as User
```

Comment:
Should we move the features up to the top?

Reply:
I added a note: 

# Features
*(The following is not a complete list of features but a work in progress to build a comprehensive feature list.)*

I think it's important that we start to have a "Feature" headline, so that contributors can start add descriptions for new features. We can use this material later on when we create docs for the dashboard.